packages/kernel-6.1/mkfs.xfs.conf

```diff
@@ -0,0 +1,5 @@
+# V5 features that were the mkfs defaults when the upstream Linux 6.1 LTS
+# kernel was released at the end of 2022.
+
+[inode]
+nrext64=0
```

Comment:
Should we keep this as the default of `1` on 6.1? If I recall correctly, isn't that when the default of `1` was introduced?

Reply:
Yes, it is. But it will show `EXPERIMENTAL Large extent counts feature in use. Use at your own risk!`. Since we are not very confident about that so we just set it 0 for now. The purpose for this change is to conveniently change it to 1 when needed.